app/components/support_interface/provider_courses_table_component.html.erb

```diff
@@ -39,7 +39,12 @@
           <% end %>
         </td>
         <% if accredited_bodies_vary? %>
-          <td class='govuk-table__cell'><%= course[:accredited_body] %></td>
+          <td class='govuk-table__cell'><%= course[:accredited_body] %>
```

Comment:
Can we link this to the provider page?

Reply:
It already links to the accredited body's provider page? https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/blob/d2bf03c502ee7d74e917ed56dc5aa00ac918eb61/app/components/support_interface/provider_courses_table_component.rb#L19